banzai_nres/utils/wavelength_utils.py

```diff
@@ -28,6 +28,16 @@ def identify_features(data, err, mask=None, nsigma=2., fwhm=6.0, **kwargs):
     return features
 
 
+def index_of_refraction_Edlen(vacuum_wavelength):
```

Comment:
Why do we need this now? The comments in the readme left me confused. I think they need more context about what we are trying to achieve and less details about the paper.

Reply:
I have left this formula here so that we can reproduce the conversion from air to vacuum if need be. In the branch I have that uses the murphy vacuum list, I have removed this function -- but I do not know exactly how long it will be until we have that branch merged in, it could be some time.

Reply:
I have updated the readme to hopefully be more clear, let me know.